smoke-tests/docker-compose.yml

```diff
@@ -5,7 +5,6 @@ x-env-base: &env_base
   HONEYCOMB_API_KEY: bogus_key
   HONEYCOMB_DATASET: bogus_dataset
   HONEYCOMB_METRICS_DATASET: bogus_dataset
-  OTEL_METRIC_EXPORT_INTERVAL: 100
```

Comment:
I think if we change this interval to 2 seconds, then the smoke tests will run quite a bit faster. I haven't tested this out myself but I figure its worth a shot!

Reply:
😭  Just tried it out, JS doesn't support this environment variable `OTEL_METRIC_EXPORT_INTERVAL` (yet?). Looks like [environment variable support is optional](https://github.com/open-telemetry/opentelemetry-specification/blob/main/spec-compliance-matrix.md#environment-variables).

I think we'll have to add this as a config option in our distro if we want to use it, maybe as an env var, until it's added upstream. It is an option on the `PeriodicExportingMetricReader` for [`exportIntervalMillis`](https://github.com/open-telemetry/opentelemetry-js/blob/2fb80ebe19c7fb16704fb39e3df1faa4845a9f8c/packages/sdk-metrics/src/export/PeriodicExportingMetricReader.ts#L87), and the default seems to be 60s.